File: integrations/aws-strands/error-code-corpus/python/code_is_a_parameter.py
"""A frame whose code arrives as a parameter and cannot be read statically."""

from ag_ui.core import EventType, RunErrorEvent


def emit(code: str) -> RunErrorEvent:
    return RunErrorEvent(
        type=EventType.RUN_ERROR,
        message="A code that arrives as a parameter.",
        code=code,
    )

File: integrations/aws-strands/error-code-corpus/typescript/code_is_a_parameter.ts
/** A frame whose code arrives as a parameter and cannot be read statically. */
import { EventType, type BaseEvent } from "@ag-ui/core";

export function emit(code: string): BaseEvent {
  return {
    type: EventType.RUN_ERROR,
    message: "A code that arrives as a parameter.",
    code,
  };
}
